app/frontend/src/components/FileInput.vue

```diff
@@ -1,35 +1,64 @@
 <template>
   <v-card class="file-input pa-2 my-2">
     <v-card-title>
-      <p>Document Generation</p>
+      <p>Document Generation Form</p>
     </v-card-title>
 
     <v-card-text>
+      <p>A successful submission requires a template file and a valid JSON object in an array.</p>
       <v-form ref="form" v-model="validFileInput">
-        <v-file-input
-          counter
-          :clearable="false"
-          label="Upload your file"
-          :mandatory="true"
-          prepend-icon="mdi-paperclip"
-          :rules="notEmpty"
-          show-size
-          v-model="files"
-        />
-        <v-text-field
-          hint="(Optional) Desired output filename"
-          label="Filename"
-          v-model="filename"
-        />
-        <v-textarea
-          auto-grow
-          hint="JSON format for key-value pairs"
-          label="Contexts"
-          :mandatory="true"
-          required
-          :rules="contextsRules"
-          v-model="contexts"
-        />
+        <v-row>
+          <v-col cols="12" md="4">
```

Comment:
Do you need the cols="12", or will it default to that below the md breakpoint? I can't remember.

Reply:
If cols="12" isn't there, it won't collapse the columns correctly on smaller screen sizes - it doesn't do that by default.